packages/node/src/indexer/api.service.ts

```diff
@@ -47,8 +48,9 @@ export class ApiService
     connectionPoolService: ConnectionPoolService<ApiPromiseConnection>,
     eventEmitter: EventEmitter2,
     private nodeConfig: NodeConfig,
+    retryManager: RetryManager,
```

Comment:
I was just meaning a util function. Adding another dependency ads a lot of unnecessary complexity. 

Its already hard enough to write tests with ApiService because we need all the connectionPool stuff. We should be able to mock an api service without all that when we want a single endpoint. But thats another issue that shouldn't be addressed here